vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -39,6 +39,7 @@ type ServerRunOptions struct {
 	MaxMutatingRequestsInFlight int
 	RequestTimeout              time.Duration
 	GoawayChance                float64
+	TerminationGoaway           bool
```

Comment:
I haven't seen the usage of this variable in your code.

Reply:
fixed.